sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseService.java

```diff
@@ -1318,8 +1318,20 @@ public void updateCompleteness(List<String> caseUuids) {
 	}
 
 	public void updateCompleteness(Case caze) {
```

Comment:
Please add a comment that this does not trigger the audit log

Reply:
Added in the code comment